src/lib/utils/hooks/SynapseAPI/useGetUserProjects.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  useInfiniteQuery,
+  UseInfiniteQueryOptions,
+} from 'react-query'
+import { SynapseClient } from '../..'
+import { SynapseClientError } from '../../SynapseClient'
+import { useSynapseContext } from '../../SynapseContext'
+import { ProjectHeaderList } from '../../synapseTypes'
+import { GetProjectsParameters } from '../../synapseTypes/GetProjectsParams'
+
+export function useGetUserProjectsInfinite(
```

Comment:
I think we already have a `useProjects` file with this hook (used in Entity Finder)

Reply:
Nvm, I see that it uses a different service. Could we put all of these hooks in the same file?